From 691e4b918771bf464a4aa73220bb0a04e9e49b6d Mon Sep 17 00:00:00 2001
From: Sugar Zhang <sugar.zhang@rock-chips.com>
Date: Wed, 8 Mar 2017 20:41:05 +0800
Subject: [PATCH] ARM64: dts: rk3399-android: add hdmi-dp drm audio

this patch register hdmi and dp as one sound card, and compatible
single hdmi audio, single dp audio or both.

Change-Id: I840e6c13f1a0c765a3a9235eb0a798011fc3bf06
Signed-off-by: Sugar Zhang <sugar.zhang@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-android.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
index bfda5a3f7e27..87e133b29d58 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
@@ -234,6 +234,13 @@
 		rockchip,uboot-charge-on = <1>;
 		rockchip,android-charge-on = <0>;
 	};
+
+	hdmi_dp_sound: hdmi-dp-sound {
+		status = "disabled";
+		compatible = "rockchip,rk3399-hdmi-dp";
+		rockchip,cpu = <&i2s2>;
+		rockchip,codec = <&hdmi>, <&cdn_dp>;
+	};
 };
 
 &vopb {
-- 
2.35.3

